client/js/stores/snippet-store.js

```diff
@@ -11,6 +11,7 @@ class SnippetStore {
     this.bindActions(RatingActions);
     this.state = {
       snippets: [],
+      currentUserSnippets: [],
```

Comment:
nope, as i've decided to add just another state to excisting store, because it uses other states(like ratings and avg) from SnippetStore, and i'd have to duplicate those in another store which isn't necessary